test/test_optim.py

```diff
@@ -2665,5 +2665,26 @@ def test_bn_update_eval_momentum(self):
 
 instantiate_parametrized_tests(TestLRScheduler)
 
+def _diff_fn(p, grad, opt_differentiable_state, opt):
+    p = p.clone()
```

Comment:
My mistake from the sample I shared, but I think you want to clone the opt_differentiable_state as well?

Reply:
done!